### ISSUE TITLE
Change storefront cache invalidation policy 

### ISSUE THREAD

**Issue by tatarincev** [VirtoCommerce/vc-platform#971]
Currently any object changes lead to change latest changes timestamp which invalidated the storefront cache. 
https://github.com/VirtoCommerce/vc-module-cache/blob/master/VirtoCommerce.CacheModule.Web/Module.cs#L104

Need to change this invalidation  behavior, only manual Cache reset operation should affect to storefront cache state.
![image](https://user-images.githubusercontent.com/7566324/27527909-30c04efc-5a4e-11e7-8d1b-2f52277396b5.png)

**Comment by Woland2k** [comment]
Also need to apply to content/cms cache, changing about us page shouldn't invalidate cache of all the other pages and themes.

**Comment by brgrz** [comment]
I'm working with Platform, editing properties, site links etc. and for each change to be reflected at storefront I have to reset cache. Is there a way to disable all caching at development time?

**Comment by tatarincev** [comment]
@brgrz Strange, Storefront should invalidate object in the cache if they are changing.  Could you please provide a concrete scenario, what data remains in the storefront cache after the changes on the platform?
FYI
You might disable cache on storefront level by change this setting 
https://github.com/VirtoCommerce/vc-storefront-core/blob/master/VirtoCommerce.Storefront/appsettings.json#L7

**Comment by brgrz** [comment]
Apparently storefront caches quite aggressively. I was seeing old main-menu even though I modified it and had to do reset cache to see changes but now that I switched caching in storefront setting to false the change was reflected immediately. Also I was changing some virtual categories and storefront collected the products in a weird way. I would see products that were removed minutes ago. That too is fixed now.
